platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/ConfigurationAdminExt.java

```diff
@@ -670,6 +682,15 @@ public Map getFactoryPidObjectClasses() {
         return metatypeList;
     }
 
+    public static boolean isPermittedToViewService(String servicePid) {
+        KeyValueCollectionPermission serviceToCheck = new KeyValueCollectionPermission(
+                "view-service.pid",
+                new KeyValuePermission("service.pid", Sets.newHashSet(servicePid)));
+        Subject subject = ThreadContext.getSubject();
```

Comment:
No, I didn't know about this method. I could probably remove the proxy from the itests since it will create a new subject for me if there isn't one bound to the ThreadContext.